runtime/vm/LayoutFFITypeHelpers.hpp

```diff
@@ -116,6 +116,10 @@ class LayoutFFITypeHelpers
 			typeCode = J9NtcPointer;
 			break;
 		case FFI_TYPE_STRUCT:
+#if defined(J9ZOS390)
+		case FFI_TYPE_STRUCT_FF: /* Fall through */
+		case FFI_TYPE_STRUCT_DD: /* Fall through */
```

Comment:
Please remove the "fall through" comments which only serve as a distraction (they're appropriate only if there's some code between one case label and the next).